### ISSUE TITLE
Adding new fonts in a Linux distro makes the tab titles change font

### ISSUE THREAD

**Issue by bennyhillthebest** [Chatterino/chatterino2#1290]
I will preface this bug report by saying that this is a very minor bug and not at all important in the grand scheme of the application. But i guess it's probably better reporting it than leaving it behind so here it is.

Bug description:
After having copied MS fonts in /usr/share/fonts/WindowsFonts and updated the fonts cache, the font in the tab title and tab selection window is different than before (and very much bolder). In the settings window the left column has the old font while the right box has the new font. The three dots selection window has not changed font.

Reproduction process:
In my case, on a Manjaro system, i copied from a different pc the W10 .ttf, .TTF and .ttc fonts and followed this Arch Wiki entry:
https://wiki.archlinux.org/index.php/Microsoft_fonts#Using_fonts_from_a_Windows_partition

Expected behaviour:
Since the tab font cannot be changed i guess the expected behaviour is that the font should not change. 

Screenshots:
https://imgur.com/a/GbyQc8R

Chatterino version:
2.1.4-beta-1
(commit 9ee286f60f2714f2b48a1343e31a8227372a1e83)

**Comment by pajlada** [comment]
Does this occur in any other Qt app you might have installed on your system? I don't believe we specify which font to use for things like those dialogs you showed, only the chat. So if the font has been modified there, I believe that might just be the font your system responds with when Qt asks what font to use

**Comment by bennyhillthebest** [comment]
In my Manjaro XFCE laptop i have a Qt settings app in which i set Regular Cantarell 10 as the default font for Qt apps. 

I went and looked at qBittorrent, nomacs and qpdfview and they are all using Cantarell 10 as instructed. Chatterino is not. 

I also tried to change to Lucida Calligraphy to see if Chatterino would "respond" to the change, and this is the result: https://imgur.com/a/y0u8jGC

If you have other questions to better troubleshoot the error don't hesitate to ask.
